### DISCUSSION TITLE
No archive template?

### DISCUSSION THREAD

**Discussion by LobsterDM** [tailpress/tailpress#106] [Q&A]
Hi,
Just wondering why there doesn't appear to be an archive template for blog posts? Is this something that can be added to the theme?
Does anyone have any template files for setting up an archive on tailpress?

Thanks

**Comment by jeffreyvr** [comment]
We have an `index.php` which WordPress uses when there is no `archive.php` present. At the moment, I'm not sure what to add to `archive.php` that is different to what index.php already has.

If you wan't to make a custom archive.php, you could use index.php as a starting point and modify from there.
